marketing_emails/email_kickoff_keynote.md

```diff
@@ -0,0 +1,39 @@
+
+- Courey Elliott: A software developer who I first meet about 2 years ago with an amazing capacity for reasoning, a wide array of interests, a constant desire to learn, and a wonderful soul.
+
+- Jurnell Cockhren: An amazingly wide and deeply learned individual inside and outside of technology. Working right at the line between software engineering, operations, and teaching, Jurnell's sharp mind, excellent sense of humor, and beautiful humanity endeared him to me from the beginning.
+
+- Sarah Guido: One of the people I greatly admire in the data science world, Sarah can take complex concepts break them down into understandable pieces with ease. Sarah's impressive engineering skills lead her to solve complex problems in data science and development alike.  With a great compassion to empower others, powerful logic, and writing, Sarah is truly an asset to our community.
+
+
+## CFP Opens Friday September 15th!
+PyTennessee 2018 is accepting all types of proposals starting Sept 15 through Nov 1, 2017. Due to the competitive nature of the selection process, we encourage prospective speakers to submit their proposals as early as possible. We’re looking forward to receiving your best proposals for tutorials and talks. Lightning talk sign-ups will be available at the registration desk the day of the event.
+
+Note: To submit a proposal, sign up or log in to your account and proceed to your account dashboard!
+
+## todo: confirm that this is true
+## Special Sponsorship Profile
+Intellovations has been the first sponsor of PyTN for all 5 years we have had the conference, and in addition to their financial support, they have been providing words of encouragement behind the scenes for all those years as well. They are truly part of the PyTN family, and for that, we are greatly appreciative and lucky to call them friends.
+
+## todo: get date for when Poung Coders begin
+## Young Coders
+We will be having young coders again this year. Tickets will be available December 15th at noon and they normally go fast.  The program will be lead by Brad Montgomery, a masterful python teacher, again this year. This free program is open to ages 12-17, and all attendees will receive take home books and a Chromebook.
+
+## Personal Note
+I add the #blame_chad because I am sure that I will miss something or communication could be better.  In light of this, I want to say thank you to those who have made the Python community so friendly and a place to turn to learn.  It allows me to step out and try to give back.  Special thanks to Jason Myers for his spirit that he poured into this event.  I jokingly add #blame_chad because of my imposter syndrome tells me that someone else could do this better.  Despite this, it is my hope to give back a bit after the Python community has given me a lot.
```

Comment:
I appreciate the sentiment here, but I think should try to keep it "on message" -- mentioning the `#blamechad` hashtag isn't a bad idea, and probably a good way to point people towards the twitter. I want you to be able to send a note like this, but I don't think one of the first emails is the place for it; it feels more "goodbye" rathe than "hello" to me.